### ISSUE TITLE
Kerberos Authentication Support

### ISSUE THREAD

**Issue by yd302148** [apache/airflow-client-python#46]
Hello,

I would like to ask if the current airflow python client supports the use of kerberos authentication? I see that the kerberos section in the README.md document is empty, and it seems that only the basic_auth function is implemented in the code https://github.com/apache/airflow-client-python/blob/99b3d15620e543cf7b01da8f076bb78ae3eb9b71/airflow_client/client/configuration.py#L398 .

Could you please give me some code examples if kerberos is available? thanks.

**Comment by potiuk** [comment]
Yes. you need to extend the class and add your own authentication if you want to use Kerberos.
